s3-proxy/nginx.conf

```diff
@@ -28,13 +28,12 @@ http {
 
         client_max_body_size 8m;
 
-        # if ($http_x_forwarded_proto = 'http') {
-        #     return 400;
-        # }
-
         location ~ ^/(?<s3_region>[^/]+)/(?<s3_bucket>[^/]+)(/(?<s3_path>.*))? {
             proxy_http_version 1.1;
             proxy_buffering off;
+            proxy_ssl_verify on;
+            proxy_ssl_verify_depth 5;
+            proxy_ssl_trusted_certificate /etc/nginx/certs.pem;
```

Comment:
I'm guessing it requires a single file, hence the `cat`

Reply:
Yep